src/main/java/com/quantumsoft/qupathcloud/pyramid/Pyramid.java

```diff
@@ -65,81 +70,97 @@ public Pyramid(List<Instance> instances) throws QuPathCloudException {
     seriesUID = instances.get(0).getSeriesInstanceUID().getValue1();
   }
 
-    /**
-     * Get downsamples double [ ].
-     *
-     * @return the double [ ]
-     */
-    public double[] getDownsamples() {
+  /**
+   * Get downsamples double [ ].
+   *
+   * @return the double [ ]
```

Comment:
The double[] is just the type I think it's better to explain what downsamples are...

Reply:
Ok